org.hlwd.biblemultithelife.yml

```diff
@@ -0,0 +1,51 @@
+app-id: org.hlwd.biblemultithelife
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: biblemultithelife
+separate-locales: false
+finish-args:
+  - --socket=x11
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --share=network
+  - --share=ipc
+  - --device=dri
+#  - --talk-name=org.freedesktop.portal.Desktop
+#  - --talk-name=org.freedesktop.portal.Documents
+  - --talk-name=org.freedesktop.portal.OpenURI
```

Comment:
you NEVER need to had dbus names for portals

Reply:
It was to try to open URI but didn't work. 
Maybe we can remove it, I changed the code to copy/paste the selected URL, so the user can open in a browser during the session.

Reply:
I let --device=dri because apparently it's a parameter to allow graphic acceleration that could be used in Flutter.